ietf/doc/tests.py

```diff
@@ -2996,6 +2996,24 @@ def test_raw_id(self):
         self.should_succeed(dict(name=draft.name, rev='00',ext='txt'))
         self.should_404(dict(name=draft.name, rev='00',ext='html'))
 
+    @override_settings(IDNITS3_BASE_URL="https://example.org/idnits3")
+    def test_nits_url(self):
+        draft = WgDraftFactory(name="draft-ietf-moon-test",rev="01", create_revisions=range(0,2))
```

Comment:
A comment as to why you're setting up this particular arrangement of revisions would be helpful